### PR TITLE
Integration PlayerJFrame and SearchVideoFrame

### PR DESCRIPTION
**Pull request by titor-oopart**

### CODE REVIEW

File: src/main/java/com/fundation/logic/main.java [hunk 1]
```diff
@@ -20,6 +20,7 @@
 public class main {
```
Comment (TheYisusGithub) [inline src/main/java/com/fundation/logic/main.java:21]: Delete this space.

File: src/main/java/com/fundation/logic/model/SearchVideo.java [hunk 2]
```diff
@@ -7,7 +7,10 @@
   * with Jalasoft.
   */
 
- package com.fundation.logic.Model;
+ package com.fundation.logic.model;
+
```
Comment (TheYisusGithub) [inline src/main/java/com/fundation/logic/model/SearchVideo.java]: delete this space in blanck.

File: src/main/java/com/fundation/logic/view/ActionPlayButton.java [hunk 3]
```diff
@@ -0,0 +1,37 @@
+/**
+ * Copyright (c) 2019 Jalasoft.
+ * This software is the confidential and proprietary information of Jalasoft.
```
Comment (MelissaRoman06) [inline src/main/java/com/fundation/logic/view/ActionPlayButton.java:4]: Empty line needed

File: src/main/java/com/fundation/logic/view/ActionPlayButton.java [hunk 4]
```diff
@@ -0,0 +1,37 @@
...
+import java.awt.event.MouseEvent;
+
+/**
+ * Implements the ButtonPlayer class
+ *
+ * @author John Salazar Pinto
+ * @version 1.0
+ */
+public class ActionPlayButton {
+    public ActionPlayButton(){
```
Comment (TheYisusGithub) [inline src/main/java/com/fundation/logic/view/ActionPlayButton.java:23]: add a space in blank about "(" and "{"

File: src/main/java/com/fundation/logic/view/SearchVideoFrame.java [hunk 5]
```diff
@@ -0,0 +1,50 @@
+/**
+ * Copyright (c) 2019 Jalasoft.
+ * This software is the confidential and proprietary information of Jalasoft.
```
Comment (MelissaRoman06) [inline src/main/java/com/fundation/logic/view/SearchVideoFrame.java:4]: Empty line needed

File: src/main/java/com/fundation/logic/view/SearchVideoPanel.java [hunk 6]
```diff
@@ -0,0 +1,163 @@
+/**
+ * Copyright (c) 2019 Jalasoft.
+ * This software is the confidential and proprietary information of Jalasoft.
```
Comment (MelissaRoman06) [inline src/main/java/com/fundation/logic/view/SearchVideoPanel.java:4]: Empty line needed

File: src/main/java/com/fundation/logic/view/SearchVideoFrame.java [hunk 7]
```diff
@@ -0,0 +1,50 @@
...
+        add(mainSearchPanel);
+        SearchVideoPanel initSearchPanel = new SearchVideoPanel();
+        contentSearchVideoPane.add(initSearchPanel);
+    }
+
+    /**
+     * This method init all setting of the components like the position
+     */
+    public void initSetting() {
+        setBounds(100, 100, 700, 700);
```
Comment (TheYisusGithub) [inline src/main/java/com/fundation/logic/view/SearchVideoFrame.java:46]: this have magical number.

File: src/main/java/com/fundation/logic/view/SearchVideoPanel.java [hunk 8]
```diff
@@ -0,0 +1,163 @@
...
+        ////Init and adds check box
+        chxNo = new JCheckBox("No");
+        add(chxNo);
+        chxYes = new JCheckBox("Yes");
+        add(chxYes);
+        //Init button
+        btnPlayer = new JButton("Player");
+        add(btnPlayer);
+    }
+
```
Comment (TheYisusGithub) [inline src/main/java/com/fundation/logic/view/SearchVideoPanel.java:111]: Delete this space.

File: src/main/java/com/fundation/logic/view/SearchVideoFrame.java [hunk 9]
```diff
@@ -0,0 +1,50 @@
...
+ * @author John Salazar Pinto
+ * @version 1.0
+ */
+public class SearchVideoFrame extends JFrame {
+    private JPanel contentSearchVideoPane;
+    public SearchVideoFrame() {
+        initComponent();
+        initSetting();
+    }
+
```
Comment (TheYisusGithub) [inline src/main/java/com/fundation/logic/view/SearchVideoFrame.java:29]: Remove this space in white.

File: src/main/java/com/fundation/logic/view/SearchVideoFrame.java [hunk 10]
```diff
@@ -0,0 +1,50 @@
...
+     */
+    public void initComponent() {
+        contentSearchVideoPane = new JPanel();
+        SearchVideoPanel mainSearchPanel = new SearchVideoPanel();
+        mainSearchPanel.setVisible(true);
+        add(mainSearchPanel);
+        SearchVideoPanel initSearchPanel = new SearchVideoPanel();
+        contentSearchVideoPane.add(initSearchPanel);
+    }
+
```
Comment (TheYisusGithub) [inline src/main/java/com/fundation/logic/view/SearchVideoFrame.java:41]: remove this space in white.

File: src/main/java/com/fundation/logic/view/SearchVideoFrame.java [hunk 11]
```diff
@@ -0,0 +1,50 @@
...
+    /**
+     * This method init all setting of the components like the position
+     */
+    public void initSetting() {
+        setBounds(100, 100, 700, 700);
+        contentSearchVideoPane.setBorder(new EmptyBorder(5, 5, 5, 5));
```
Comment (TheYisusGithub) [inline src/main/java/com/fundation/logic/view/SearchVideoFrame.java:47]: it has magical number.

**Pull request merged**